03.asynchronous/db_operations.js

```diff
@@ -0,0 +1,54 @@
+import sqlite3 from "sqlite3";
+
+export class Database {
+  constructor(driver) {
+    this.db = driver;
+  }
+
+  run(sql, params = []) {
```

Comment:
`params` にデフォルト引数を設定している理由を教えてください。

Reply:
@cafedomancer 
node-sqlite3のAPIドキュメント内に`In case you want to keep the callback as the 3rd parameter, you should set param to "[]" ( Empty Array ) as per issue #116`という説明があったので、`params`に空配列をデフォルト引数に置いていました。
↓↓↓
[issue #116](https://github.com/mapbox/node-sqlite3/issues/116)